.gitignore

```diff
@@ -0,0 +1,50 @@
+# Maven
+target/
+pom.xml.tag
+pom.xml.releaseBackup
+pom.xml.versionsBackup
+pom.xml.next
+release.properties
+dependency-reduced-pom.xml
+
+# Compiled class file
+*.class
+
+# Log file
+*.log
+*.log*
+
+# Package Files #
+*.jar
```

Comment:
ezeket is nem gitignore szinten kellene toltani hanem commit tiltassal

Reply:
kivettem